packages/gasket-engine/test/exec.test.js

```diff
@@ -81,4 +137,28 @@ describe('The exec method', () => {
 
     await expect(engine.exec('mock')).rejects.toThrow(Error);
   });
+
+  it('should await if ordered', async function () {
```

Comment:
```suggestion
  it('should await if ordered', async () => {
```
Small change to conform to how the rest of the tests are written.